docs/ocis/deployment/ocis_oc10_backend.md

```diff
@@ -286,4 +360,22 @@ ADD apps/graphapi-0.1.0.tar.gz /var/www/owncloud/apps/
 
 The build is triggered by the terminal command `docker-compose build` from the root folder.
 
-Constraints: In this setup it's mandatory that the user has an email adress set in oc10.
+Constraints: In this setup it's mandatory that the user has an email address set and is assigned to at least one group in oc10.
+Especially the default admin user doesn't have an email assigned. If your admin user doesn't have an email address, yet, please
+set one: `docker-compose exec owncloud occ user:modify admin email "admin@example.org"`
+
+## Local deployment
+
+In case you want to start the bridge setup on your local development machine, there are a few steps necessary:
```

Comment:
ok so this is a description of a production deployment and this section now tells the reader how to use it on the local machine. Or is this about a remote deployment vs a local deployment?

In any case I think it would make sense to clarify the deployment scenarios in `## Remote deployment` (for testing), `## Local deployment` (for development) and `## Production deployment`. Or maybe `## <type> scenario`?